### ISSUE TITLE
View finder not working with IE 11 on a touch screen

### ISSUE THREAD

**Issue by pgrm** [novus/nvd3#1051]
Hi, the view finder doesn't seem to work with IE 11 using the touchscreen on a windows tablet device. It works find using the mouse, and when tested in Firefox it works both with the mouse, as wells as the touch screen. We've tested it with the example page:  http://nvd3.org/examples/lineWithFocus.html

Do you have any thoughts on this which might help us?
In worst case, if it couldn't be solved, do you know how we could deactivate and hide it, if the current device is an IE 11 (or smaller) browser, without a mouse? I guess the *without a mouse* part will be the hardest.

**Comment by pgrm** [comment]
Ok @detecting touch based input, seems like the folks at Modernizr have thought about it and discarded it: https://github.com/Modernizr/Modernizr/issues/869

The only more or less viable solution suggested would be the use of [PointerEvents](https://msdn.microsoft.com/library/hh772103(v=vs.85).aspx) in IE 10 and 11 (that's what I actually care about) and probably they meant to use the [pointerType](https://msdn.microsoft.com/en-us/library/hh772359(v=vs.85).aspx) with which I could detect, if there is ever a touch input and in that case set a global flag and hide the view finder? It's a terrible solution (I believe) but maybe the only one, if you don't know how else it could get fixed?

**Comment by pgrm** [comment]
Ok, in case the view finder simply uses D3 brushing, the problem seems to be in D3, I've reported it here: https://github.com/mbostock/d3/issues/2446

In case it doesn't use brushing, than there are 2 problems around.

**Comment by liquidpele** [comment]
Hmm... unfortunately I don't have a device I can verify this bug with.. @robinfhu or anyone else can you?

**Comment by robinfhu** [comment]
Yeah I don't have a Windows tablet.  In general, I'm not sure our mobile support is that strong.

**Comment by liquidpele** [comment]
Well, I've tested on ipad/safari and android/chrome at least, stuff seems to work with those for me.  As of a few months ago anyway.  I did that in response to this:

https://github.com/nvd3-community/nvd3/pull/19

Never did merge it in since I couldn't reproduce any problems that it would fix...

**Comment by pgrm** [comment]
as I said - firefox/windows works fine, I've also used it for a long time with chrome on my nexus 5 and that still works, so it might be an IE 11 bug. Wouldn't be the first one I came across. Next week I'll try it with Windows 10 beta and the corresponding IE

**Comment by liquidpele** [comment]
Yup, was just replying to Robin about our mobile support not being strong :)
